### PR TITLE
1.13.0 changelog

### PR DESCRIPTION
**Pull request by trabus**
In prep for release. Need to merge #4391 before release.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,84 @@
 # ember-cli Changelog
 
+### 1.13.0
+
+The following changes are required if you are upgrading from the previous
+version:
+
+- Users
+  + Upgrade your project's ember-cli version - [docs](http://www.ember-cli.com/#project-update)
```
Comment (stefanpenner) [inline CHANGELOG.md]: I believe the brocfile.json has changed

cc @chadhietala

Comment (trabus) [inline CHANGELOG.md]: should we mention the addition of `ember-cli-build.js` as well?

Comment (stefanpenner) [inline CHANGELOG.md]: ya i think so.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,85 @@
...
+- Addon Developers
+  + `Brocfile.js` has been deprecated in favor of `ember-cli-build.js`. See [TRANSITION.md](https://github.com/ember-cli/ember-cli/blob/master/TRANSITION.md) for details on how to transition your `Brocfile.js` code to `ember-cli-build.js`.
+  + Blueprints can now be generated into the `tests/dummy/app` folder with the `--dummy` flag.
+  + Scoped npm dependencies are now supported.
+- Core Contributors
+  + fs.existsSync is deprecated, use exists-sync instead.
+
+#### Community Contributions
+- [#4391](https://github.com/ember-cli/ember-cli/pull/4391) Update ember-data to 1.13.4
+- [#4300](https://github.com/ember-cli/ember-cli/pull/4300) Update Ember to v1.13.1. [@rwjblue](https://github.com/rwjblue)
```
Comment (sly7-7) [inline CHANGELOG.md]: maybe we should bump to the latest version as well ?

Comment (trabus) [inline CHANGELOG.md]: Latest ember version? We're on [1.13.3](https://github.com/ember-cli/ember-cli/blob/master/blueprints/app/files/bower.json#L4)

Comment (sly7-7) [inline CHANGELOG.md]: I'm sorry, I don't even have an eye to the actual file, just looking at this changelog entry, and thought it was the last one.

Comment (trabus) [inline CHANGELOG.md]: No problem, there were 4 ember updates scattered throughout. I should probably thin the changelog to just the latest version for clarity.

**Pull request merged**